lib/ProductOpener/Config_off.pm

```diff
@@ -91,6 +92,39 @@ use vars @EXPORT_OK ; # no 'my' keyword for these
 
 use ProductOpener::Config2;
 
+# define the normalization applied to change a string to a tag id (in particular for taxonomies)
+# tag ids are also used in URLs.
+
+# unaccent:
+# - useful when accents are sometimes ommited (e.g. in French accents are often not present on capital letters),
+# either in print, or when typed by users.
+# - dangerous if different words (in the same context like ingredients or category names) have the same unaccented form
+# lowercase:
+# - useful when the same word appears in lowercase, with a first capital letter, or in all caps.
+
+%string_normalization_for_lang = (
+	# no_language is used for strings that are not in a specific language (e.g. user names)
+	no_language => {
+		unaccent => 1,
+		lowercase => 1,
+	},
+	# default is used for languages that do not have specified values
+	default => {
+		unaccent => 1,
```

Comment:
As per the discussion on openfoodfacts/openfoodfacts-server#458, the default should probably be 0.
```suggestion
		unaccent => 0,
```

Reply:
My thinking was to minimize changes to the current behaviour, and test everything out first in one language (German). We can make the default 0, but in that case I will add unaccent=>1 for English, Spanish, Italian and Portuguese.

Reply:
in addition to French